tests/integration/targets/database/aliases

```diff
@@ -0,0 +1,2 @@
+context/target
```

Comment:
can be left out (will be detected as a module and known to be in target context) but mentioning more as an informational thing, can leave it